lib/bugsnag/configuration.rb

```diff
@@ -173,6 +177,13 @@ def should_notify_release_stage?
       @release_stage.nil? || @notify_release_stages.nil? || @notify_release_stages.include?(@release_stage)
     end
 
+    ##
+    # Regex used to mark stacktrace lines as within the project. Lines marked
+    # as "out of project" will only appear in the full trace.
+    def vendor_path
+      @vendor_path || Bugsnag::Stacktrace::VENDOR_PATH
```

Comment:
Consider moving/renaming `Bugsnag::Stacktrace::VENDOR_PATH` to `Bugsnag::Configuration::DEFAULT_VENDOR_PATH`, as it is now a default, and it might be easier for users to find/discover this constant in `Bugsnag::Configuration` in case they need to use this regular expression to add to it for a new vendor path regular expression (maybe using something like `Regexp.union`).